src/asymmetric_cipher.c

```diff
@@ -3,6 +3,7 @@
 
 #include "provider.h"
 #include <string.h>
+#include <strings.h>
```

Comment:
this came from the strcasecmp) usage right?
If so drop from here and the other two files

Reply:
Yeah, it had the extra commits from other branches which are now merged, so I dropped this.